pkg/operator/operator.go

```diff
@@ -100,16 +93,10 @@ func (o *Operator) Run() error {
 	}
 	pc := controller.NewProvisionController(
```

Comment:
do all these fields now have defaults inside the `ProvisionController` or are they completely obsolete?